app/config.go

```diff
@@ -34,5 +36,7 @@ func NewWorkflowControllerConfig() *Config {
 // AddFlags add cobra flags to populate Config
 func (c *Config) AddFlags(fs *pflag.FlagSet) {
 	fs.StringVar(&c.KubeConfigFile, "kubeconfig", c.KubeConfigFile, "Location of kubecfg file for access to kubernetes master service")
-	fs.StringVar(&c.ListenHTTPAddr, "addr", "0.0.0.0:8086", "listen address of the http server which serves kubernetes probes and prometheus endpoints")
+	fs.StringVar(&c.ListenHTTPAddr, "addr", "0.0.0.0:8086", "Listen address of the http server which serves kubernetes probes and prometheus endpoints")
+	fs.BoolVar(&c.EnableCronWorkflow, "enable-cron-workflow", false, "Enable CronWorkflow controller")
```

Comment:
Why do you want to propose the possibility to activate or not CronWorkflow?

Reply:
hmmm... At the beginning I thought it was a good idea. I think you're right. I'm going to enable by default.